File: example/src/main/java/com/ice/restring/example/MainActivity.java
package com.ice.restring.example;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ((TextView) findViewById(R.id.text_view2)).setText(R.string.subtitle);
    }
}

File: example/src/main/java/com/b3nedikt/restring/example/MainActivity.kt
package com.b3nedikt.restring.example

import android.os.Bundle
import com.b3nedikt.restring.RestringLocale
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*

class MainActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Swiss german
        RestringLocale.currentLocale = Locale("de", "CH")

        setContentView(R.layout.activity_main)

        titleTextView.text = getText(R.string.title)
    }
}
